### DISCUSSION TITLE
NodeJS http module?

### DISCUSSION THREAD

**Discussion by HimanshuMishir** [uNetworking/uWebSockets.js#838] [Q&A] [answered]
How to get query parameters in  uWebSockets.js  http get method?
like http://localhost:5000/countOfConnectedUsers/?passkey=somerandomtext
how to get the value of passkey?

**Comment by uNetworkingAB** [comment, marked as answer]
https://unetworking.github.io/uWebSockets.js/generated/interfaces/HttpRequest.html#getQuery
